### DISCUSSION TITLE
Deployment from multiple repos to the same web app

### DISCUSSION THREAD

**Discussion by lukaszch** [Azure/static-web-apps#924] [General]
We keep our public documentation web pages within multiple repos to separate their content and make them more manageable.
However, we want to publish them into one Azure Static Web App under different subfolders, e.g.: /example.com/doc-first and /example.com/doc-second.
Is it possible?
I cannot find any documentation on how to do this from GitHub Actions.

**Comment by thomasgauvin** [comment]
Hi @lukaszch, it is not possible to partially deploy to Static Web Apps routes. To do this, you would have to build & package your Static Web Apps in one app & deploy this.
